content/02.introduction.md

```diff
@@ -1,7 +1,9 @@
 
-To address these shortcomings, we have completely re-implemented the latest version of the EAGER pipeline in Nextflow (@doi:10.1038/nbt.3820) (a domain-specific-language (DSL) designed for the construction of omics analysis pipelines), introduced new features, and more flexible pipeline configurations. In addition, the newly named pipeline - nf-core/eager - has been developed in the context of the nf-core community framework (@doi:10.1038/s41587-020-0439-x), which enforces strict guidelines for best-practises in software development.
+Two previously published and commonly used pipelines in the field are PALEOMIX [@doi:10.1038/nprot.2014.063] and EAGER [@doi:10.1186/s13059-016-0918-z]. These two pipelines take a similar approach to link together standard tools used for Illumina NGS data processing (sequencing quality control, sequencing adapter removal/and or paired-end read merging, mapping of reads to a reference genome, genotyping, etc.), but with a specific focus on tools that are designed for or well-suited for aDNA (such as bwa aln that works well on ultra-short molecules [@doi:10.1093/bioinformatics/btp324] and mapDamage [@doi:10.1093/bioinformatics/btr347] for aDNA characteristics evaluation). Yet, neither of these pipelines have had major updates to bring them in-line with current routine aDNA analyses. Metagenomic screening of off-target genomic reads for pathogens or microbiomes [@doi:10.1038/ng.2906, @doi:10.1038/nature21674] has become particularly common, given it's role in revealing widespread infectious disease and possible epidemics that had previously been undetected in the archaeological record [@doi:10.1016/j.cell.2015.10.009, @doi:10.1016/j.cub.2017.10.025, @doi:10.1038/s41586-018-0097-z, @doi:10.7554/eLife.36666]. Without easy access to the latest field-established analytical routines, aDNA studies from groups new to the field risk being published without the necessary quality control checks that ensure aDNA authenticity and without yielding the full range of possibilities from their data.
+
+To address these shortcomings, we have completely re-implemented the latest version of the EAGER pipeline in Nextflow [@doi:10.1038/nbt.3820] (a domain-specific-language (DSL), specifically designed for the construction of omics analysis pipelines), introduced new features, and more flexible pipeline configurations. In addition, the newly named pipeline - nf-core/eager - has been developed in the context of the nf-core community framework [@doi:10.1038/s41587-020-0439-x], which enforces strict guidelines for best-practises in software development.
```

Comment:
> best-practises in software development.

best-practises in Nextflow software development.

Reply:
Stuff like automated CI is generic!